.travis.yml

```diff
@@ -2,22 +2,20 @@ language: go
 go:
   - 1.5.3
 script:
-  - go test -v ./... 
+  - go test -v ./...
 before_deploy:
   - go get github.com/inconshreveable/mousetrap # temporary fix for travis build breaks for windows due to cobra dependency on mousetrap
-  - GOOS=darwin GOARCH=amd64 go build -o parse -v github.com/ParsePlatform/parse-cli 
-  - GOOS=windows GOARCH=386 go build -o parse.exe -v github.com/ParsePlatform/parse-cli
-  - GOOS=linux GOARCH=386 go build -o parse_linux -v github.com/ParsePlatform/parse-cli
-  - GOOS=linux GOARCH=arm go build -o parse_linux_arm -v github.com/ParsePlatform/parse-cli 
+  - GOOS=darwin GOARCH=amd64 go build -o back4app -v github.com/back4app/parse-cli
+  - GOOS=windows GOARCH=386 go build -o back4app.exe -v github.com/back4app/parse-cli
+  - GOOS=linux GOARCH=386 go build -o back4app_linux -v github.com/back4app/parse-cli
+  - GOOS=linux GOARCH=arm go build -o back4app_linux_arm -v github.com/back4app/parse-cli
 deploy:
   provider: releases
-  api_key:
```

Comment:
We must set `api_key` to be able to deploy through travis.

https://docs.travis-ci.com/user/encryption-keys/
https://docs.travis-ci.com/user/deployment